shark_turbine/kernel/ops/wave_ops.py

```diff
@@ -198,6 +206,25 @@ def copy(self, new_name: Optional[str] = None) -> Self:
             new_node.name = new_name
         return get_custom(new_node)
 
+    def copy_to_new_graph(
+        self, new_graph: fx.Graph, new_name: Optional[str] = None
+    ) -> Self:
+        """Returns a duplicate of this node."""
+        new_node = new_graph.node_copy(self.fx_node)
+        new_node.tkw_op = self
+        if new_name:
+            new_node.name = new_name
+        return get_custom(new_node)
```

Comment:
Would it make sense to add the `new_graph` as optional argument to our existing `copy` function above?

Reply:
That's also what I was wondering but decided to keep it separate to hear back from you during review. Thanks for the clarification. I will update that function and remove this new one.